generic/csp/metadata.yaml

```diff
@@ -167,3 +167,47 @@ policyRules:
     tldr: Audit device directory for enhanced security
```

Comment:
in this file `generic/csp/metadata.yaml` there is a repeated `refs:` added in [line #62 ](https://github.com/kubearmor/policy-templates/blob/0cf61034c24da28713066d91e3fb0ed8206ef20c/generic/csp/metadata.yaml#L62)


can you remove that and check if the yaml is valid with some tools like `yq`